extensions/vscode-api-tests/package.json

```diff
@@ -41,13 +41,12 @@
 		}
 	},
 	"scripts": {
-		"compile": "node ./node_modules/vscode/bin/compile -watch -p ./",
-		"vscode:prepublish": "node ../../node_modules/gulp/bin/gulp.js --gulpfile ../../build/gulpfile.extensions.js compile-extension:vscode-api-tests ./tsconfig.json"
+		"compile": "gulp compile-extension:vscode-api-tests",
+		"watch": "gulp watch-extension:vscode-api-tests"
```

Comment:
Thanks for the clarifications. I still do think there is value in keeping this consistent across our builtin extensions and not ending up with an extra unused typescript instance after you run npm install